artifactory_test.go

```diff
@@ -3345,6 +3367,7 @@ func createRandomReposName() {
 	tests.Repo1 += "-" + timestamp
 	tests.Repo2 += "-" + timestamp
 	tests.Repo1And2 += "-" + timestamp
+	tests.Repo1And2Placeholder += "-" + timestamp
```

Comment:
Shouldn't this new repo be added to the GetNonVirtualRepositories()? I can't see the code which creates this repo.

Reply:
This test uses the existing **'Repo1'** and **'Repo2'** which are created with **GetNonVirtualRepositories()**.
I added the **'Repo1And2Placeholder'** to **artifactoryconsts.go**, in order to get the generated timestamp which affects the repository names.
Then the 'Repo1And2Placeholder' is used as the 'pattern' provided to the CLI download command.